azurelinuxagent/ga/update.py

```diff
@@ -422,6 +422,28 @@ def _emit_restart_event(self):
 
         return
 
+    @staticmethod
+    def _emit_changes_in_default_configuration():
+        try:
+            def log_if_int_changed_from_default(name, current):
+                default = conf.get_int_default_value(name)
+                if default != current:
+                    msg = "{0} changed from its default; new value: {1}".format(name, current)
```

Comment:
Suggestion, log the default value here too?

Reply:
i had it initially; too verbose, so i removed it